src/ducks/transactions/TransactionsPage.jsx

```diff
@@ -172,12 +184,21 @@ class TransactionsPage extends Component {
         limitMin = 0
       }
     }
-    this.setState({
-      limitMin: limitMin,
-      limitMax: limitMin + 5,
-      currentMonth: month,
-      infiniteScrollTop: false
-    })
+    this.setState(
+      {
+        limitMin: limitMin,
+        limitMax: limitMin + 10,
+        currentMonth: month,
+        infiniteScrollTop: false
+      },
+      () => {
+        // need to scroll past the LoadMore button
+        if (onIOS()) {
+          const scrollTo = limitMin > 0 ? 65 : 0
```

Comment:
Does that mean that if for any reason we change the style of the LoadMore button, we also need to change the value here ? Is it possible to retrieve the button height dynamically ?

Reply:
Yes, will try to do that :)

Reply:
But it's not blocking for me. It's just because I'm 100% sure that we will forget to update it if we update the LoadMore button :p